man/man8/zpool.8

```diff
@@ -115,6 +115,11 @@
 .Ar pool Ns | Ns Ar id
 .Op Ar newpool Oo Fl t Oc
 .Nm
+.Cm import
+.Fl C Ar configfile
```

Comment:
Elsewhere in the manpage, we refer to this as a `cachefile` (e.g. `zpool import -c cachefile`).  I think we should use that term throughout, it will make it more clear that the file can then be used with `zpool import -c` (or zdb -c)

Reply:
good point. fixed